EncounterScript_StreamlabsSystem.py

```diff
@@ -208,6 +211,105 @@ class Item:
     offence = 0
     defence = 0
 
+# ---------------------------------------
+# Functions used for the Gameplay Log File
+# ---------------------------------------
+
+def CreateGameplayLogFile():
+    create = open(LogFile, "w+")
+    create.close()
+    data2 = {}
+    unformattedDate = date.today()
```

Comment:
Is it possible to get the date and format it in one line (where it's assigned to `arr['date']`), rather than the two lines of code

Reply:
Reduced to one line.